### PR TITLE
[REF] Rename html surface func to make them private

### LINKED ISSUE #3500: Renaming plotting.html_surface.one_mesh_info() and plotting.html_surface.full_brain_info()
This is a follow-up issue from #3173.

`plotting.html_surface.one_mesh_info()` and `plotting.html_surface.full_brain_info()` should be renamed to `plotting.html_surface._one_mesh_info()` and `plotting.html_surface._full_brain_info()` respectively so that it becomes explicit that they're not in the public API.

### PR DESCRIPTION
**Pull request by Rahul-Brito**
Closes #3500. Renamed files and also changed comment in one example, didn't seem to appear elsewhere.

Changes proposed in this pull request:

- Rename `one_mesh_info` and `full_brain_info`
- Add first-time contributor info

### CODE REVIEW

**Comment by Remi-Gau** [comment]
converting to a draft PR for now as the title still mentions WIP. :-)

**Comment by htwangtw** [comment]
@Rahul-Brito if you need help to setup development environment to make the process easier, please let us know

**Comment by Rahul-Brito** [comment]
Thank you @htwangtw! I think I have it set up. I also found this PR https://github.com/nilearn/nilearn/pull/3505 from the last time sometime tried to do this and ran into similar issues as me, so I'm pretty sure I know what I need to do! Thanks for your patience :)

File: nilearn/plotting/html_surface.py [hunk 1]
```diff
@@ -190,6 +204,19 @@ def full_brain_info(volume_img, mesh='fsaverage5', threshold=None,
     info['colorscale'] = colors['colors']
     return info
 
+def full_brain_info(volume_img, mesh='fsaverage5', threshold=None,
+                     cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
+                     bg_on_data=False, darkness=.7,
+                     vmax=None, vmin=None, vol_to_surf_kwargs={}):
+    warnings.warn(category=DeprecationWarning, 
+                  message="full_brain_info is a private function and is renamed to"
```
Comment (bthirion) [inline nilearn/plotting/html_surface.py:212]: Please organize the line cleanly.

File: nilearn/plotting/tests/test_html_surface.py [hunk 2]
```diff
@@ -98,11 +98,22 @@ def test_one_mesh_info():
     assert not info['full_brain_mesh']
     check_colors(info['colorscale'])
 
+    #public one_mesh_info was made private _one_mesh_info
+    #a new public one_mesh_info was made that provides a warnng and runs the private method
```
Comment (bthirion) [inline nilearn/plotting/tests/test_html_surface.py:102]:
```suggestion
    #a new public one_mesh_info was made that provides a warning and runs the private method
```

**Review by bthirion — COMMENTED** [review]
Thx ! 
Simply take care of the line formatting.

**Comment by Rahul-Brito** [comment]
I think Black updated the spacing (or maybe @Remi-Gau did). Do I need to do anything else on this one?

**Comment by ymzayek** [comment]
@Rahul-Brito the changes are good. I will fix the formatting issues since the difficulty here is due partly to automatic formatting not having been applied to these files

**Review by Remi-Gau — APPROVED** [review]
LGTM

**Comment by Remi-Gau** [comment]
Thanks for taking care of this @Rahul-Brito

**Pull request merged**